app/src/main/res/values/strings.xml

```diff
@@ -481,12 +481,28 @@
     <string name="settings">Settings</string>
     <string name="language">Language</string>
     <string name="choose_language">Choose your language</string>
+    <string name="choose_theme">Choose your theme</string>
     <string name="notification">Notifications</string>
     <string name="base_url">Base URL</string>
     <string name="tenant">Tenant</string>
     <string name="no_saving_account">Saving Accounts</string>
     <string name="no_loan_account">Loan Accounts</string>
     <string name="no_sharing_account">Sharing Accounts</string>
+    <string name="theme">Theme</string>
+    <string name="ui">UI</string>
+    <string name="dark_mode_def_value">MODE_NIGHT_FOLLOW_SYSTEM</string>
+
+    <string-array name="mode_entries"  translatable="false">
```

Comment:
mode_entries are UI related strings, please make it `translatable="true"` or add items like ` <item>@string/light</item>`

Reply:
ok